problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -79,7 +79,15 @@ function MentoringWithStepsBlock(runtime, element) {
         $.post(handlerUrl, JSON.stringify({}))
             .success(function(response) {
                 attemptsDOM.data('num_attempts', response.num_attempts);
-                // Now that relevant info is up-to-date, get the latest grade
+                publishAttempt();
+            });
+    }
+
+    function publishAttempt() {
+        var handlerUrl = runtime.handlerUrl(element, 'publish_attempt');
+        $.post(handlerUrl, JSON.stringify({}))
```

Comment:
@itsjeyd do we have an integration test for this? There's no easy way (if *any* way exists) to test js parts of XBlocks so far, but it is possible to create an integration tests that would test sending grades when user answered all the questions